lib/templates.ts

```diff
@@ -1424,3 +1432,45 @@ export const checkIfClosed = async (formId: string) => {
     return null;
   }
 };
+const checkForBetaComponents = async (elements: FormElement[]) => {
```

Comment:
Can we use the ` checkForBetaComponents ` from the store level here?  

Looking to have 1 "checker" function vs the current 2

Reply:
We could.  I was just trying to separate the concerns between client and sever since the Store is completely client based.  What I could do is pull the betaCheck out of the Store folder structure and find a safe place for lib's that support both client and server.